File: hidden-api/src/main/java/android/text/format/FormatterHidden.java
package android.text.format;

import android.content.Context;


import androidx.annotation.Nullable;

import dev.rikka.tools.refine.RefineAs;

@RefineAs(Formatter.class)
public class FormatterHidden {
  public static int FLAG_SI_UNITS;

  public static int FLAG_IEC_UNITS;

  public static String formatFileSize(@Nullable Context context, long sizeBytes, int flags) {
    throw new RuntimeException("Stub");
  }
}

File: hidden-api/src/main/java/android/text/format/FormatterHidden.kt
package android.text.format

import android.content.Context
import dev.rikka.tools.refine.RefineAs

@RefineAs(Formatter::class)
class FormatterHidden {
  companion object {
    @JvmField
    var FLAG_SI_UNITS: Int = 0

    @JvmField
    var FLAG_IEC_UNITS: Int = 0

    @JvmStatic
    fun formatFileSize(context: Context?, sizeBytes: Long, flags: Int): String {
      throw RuntimeException("Stub")
    }
  }
}